lib/ansible/modules/group.py

```diff
@@ -57,6 +57,15 @@
         type: bool
         default: no
         version_added: "2.8"
+    prefix_path:
+        description:
+            - Prefix path to use when updating groups. The directory where this variable is pointing to needs to contain
+              at least the file C(etc/group). If this file is empty, then the caller must ensure that the required
+              I(root) and system groups are created.
+            - Not supported on macOS or BusyBox distributions, Ubuntu older than (or equal to) version 18 and RedHat
+              distributions older than (or equal to) 6.
+            - Mutually exclusive with C(local)
+        type: path
```

Comment:
@samdoran i did not add it here and just stumbled upon some documentation, but should a `version_added: "2.12"` be added here? Am I correct that a changelog fragment is not necessary then?

Reply:
>  but should a version_added: "2.12" be added here

Yes.

> how does it then work with backporting new additions?

We do not backport new features, only bug and security fixes.

Reply:
> > but should a version_added: "2.12" be added here
> 
> Yes.

Done

> 
> > how does it then work with backporting new additions?
> 
> We do not backport new features, only bug and security fixes.

And understood!